Code/Engine/Core/ResourceManager/ResourceHandle.h

```diff
@@ -240,7 +240,7 @@ class ezTypedResourceHandle
 template <typename T>
 struct ezHashHelper<ezTypedResourceHandle<T>>
 {
-  EZ_ALWAYS_INLINE static ezUInt32 Hash(const ezTypedResourceHandle<T>& value) { return value.GetResourceIDHash(); }
+  EZ_ALWAYS_INLINE static ezUInt32 Hash(const ezTypedResourceHandle<T>& value) { return ezHashingUtils::StringHashTo32(value.GetResourceIDHash()); }
```

Comment:
why is this necessary ?

Reply:
GetResourceIDHash returns a 64bit hash but the Hash function for the HashTable etc. expects a 32bit hash. I guess we never used a resource handle as key in a hashtable before so this was never a problem.